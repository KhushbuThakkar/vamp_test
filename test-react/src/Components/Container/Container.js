import React from "react";
import PropTypes from "prop-types";
import Box from "../Box/Box";

const Container = (props) => (
  <Box
    sx={{
      maxWidth: "60%",
      mx: "auto",
      px: 3,
    }}
  >
    {props.children}
  </Box>
);

Container.propTypes = {
  children: PropTypes.element.isRequired,
};

export default Container;
