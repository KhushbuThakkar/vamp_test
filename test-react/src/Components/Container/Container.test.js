import React from "react";
import renderer from "react-test-renderer";
import Container from "./Container";

const testComponent = () => {
  return <p>Hello from test component</p>;
};
it("render correctly container component", () => {
  const component = renderer.create(
    <Container>
      <testComponent />
    </Container>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
