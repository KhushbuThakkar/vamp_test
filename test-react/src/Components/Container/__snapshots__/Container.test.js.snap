// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly container component 1`] = `
<div
  className="css-fm9nlt"
>
  <testComponent />
</div>
`;
